object_manager.go

```diff
@@ -387,3 +387,20 @@ func (objMgr *ObjectManager) CreateMultiObject(req *MultiRequest) ([]map[string]
 
 	return result, nil
 }
+
+/// GetUpgradeStatus returns the status of the grid information of a map
+func (objMgr *ObjectManager) GetUpgradeStatus(type_ string) ([]map[string]interface{}, error) {
```

Comment:
why type_ and not only type?

Reply:
`type` is go's keyword, to avoid confusion used type_

Reply:
Please use something more self explanatory, e.g.
If `type` refers to type of upgrade, use `upgradeType`
If it refers to type of status, use `statusType`

Reply:
Done.

Reply:
According to WAPI doc (see next comment), the type refers to "upgrade status type".  So, I recommend changing it to `statusType`

Reply:
Sure.